### DISCUSSION TITLE
Welcome to antminer-monitor Discussions!

### DISCUSSION THREAD

**Discussion by anselal** [anselal/antminer-monitor#196] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by smwbev** [comment]
Greetings.
Can you please tell me how to add fresh miners Antminer
For example L7

**Reply by anselal** [reply]
Check https://github.com/anselal/antminer-monitor/discussions/197
